oap-ws-account/src/main/java/oap/ws/account/OrganizationWS.java

```diff
@@ -385,7 +387,7 @@ protected ValidationErrors validateCreateApikey( UserData loggedUser, String org
     private ValidationErrors validateEmailOrganizationAccess( String organizationId, String email ) {
         return accounts.getUser( email )
             .filter( u -> !u.belongsToOrganization( organizationId ) && u.getRole( SYSTEM ).isEmpty() )
-            .map( u -> error( FORBIDDEN, "User belongs to other organization  " + email + "::" + organizationId ) )
+            .map( u -> error( FORBIDDEN, "User" + email + "does not belong to organization " + organizationId ) )
```

Comment:
add space between User and email (otherwise it will be Userajkdfshgkjasdhf@mail.com

Reply:
done

Reply:
do not see the fix in
"User" + email + "does not belong to organization "

Reply:
very strange
here is my fix
![Uploading Screenshot 2024-01-03 at 13.39.52.png…]()